### ISSUE TITLE
[Python] Codegen for definition with no properties leads to missing models

### ISSUE THREAD

**Issue by mbohlool** [swagger-api/swagger-codegen#5400]
Similar to #4804 

Models with no property are valid according to [swagger 2.0](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#schemaObject) spec:

```
...
Unlike previous versions of Swagger, Schema definitions can be used to describe primitive and arrays as well.
...
```
but code generator does not support them. My suggestion is to create classes that extends primitive types in this case. e.g., for something like this:

```
"MyString": {
"Description": "My string description",
"type": "string",
}
```

generator should generate something similar to:

```
...
class MyString(string):
   """My string description"""
   pass
...
```

Also if there is no type, we should assume it is an free range schema and class should extend "object".

**Comment by wing328** [comment]
@mbohlool thanks for reporting the issue. When I clicked on https://github.com/kubernetes-incubator/client-python/blob/master/kubernetes/docs/V1Patch.md, I got a 404.

Is there a spec we can more easily reproduce the issue?

cc @scottrw93 @baartosz @frol @taxpon @scop

**Comment by mbohlool** [comment]
@wing328 That is exactly the issue. We have a v1.Patch that is not resulted in generating a model.

https://github.com/kubernetes-incubator/client-python/blob/e69d4fe204b98f7d7ee3ada3996b4f5fbceae5fe/scripts/swagger.json

I've linked to a specific commit because I am implementing a workaround for now and this file will be changed soon. But in the commit I linked about, the file has the model but generated client does not.

**Comment by mbohlool** [comment]
You can also look at `v1.Time` and `resource.Quantity` models that has the same problem in the above spec.

**Comment by taxpon** [comment]
@wing328 Do you know how other language generators are working regarding this issue?

**Comment by wing328** [comment]
@taxpon not that I'm aware of. It looks to me it's an issue with Swagger Parser skipping models with no properties.

@mbohlool I found the following:
```
parameters: [
{
  name: "body",
  in: "body",
  required: true,
  schema: {
    $ref: "#/definitions/v1.Patch"
  }
}
],
```
Can you provide an example of what the JSON body looks like?

A workaround is to define a dummy property (e.g. id)

**Comment by mbohlool** [comment]
@wing328 That is a free json object. The server interpret it based on the context. e.g. if it is a Patch on `Pod`, then it would look for `Pod`'s fields to patch. This could be better designed but still a valid swagger.

**Comment by wing328** [comment]
I think it should be described with just "type: object" instead if it's an arbitrary JSON (but I don't think the Python API client supports that yet in the body parameter)

Ref: http://stackoverflow.com/questions/35782175/how-to-receive-a-dynamic-response-in-a-swagger-spec/35791870#35791870

**Comment by mbohlool** [comment]
Yes, I've fixed that. Python client does support that in body if it is inlined (not a model) but when it is a model, it is not supported. I've inlined that in my new spec for now but it would be nice if we can fix python client to accept primitive models.

**Comment by wing328** [comment]
> Do you know how other language generators are working regarding this issue?
@taxpon  I think it's more of an issue with the Swagger Parser itself so all generators should have the same issue. Please try the workaround to see if it works for you.

We'll use https://github.com/swagger-api/swagger-codegen/issues/4804 for tracking the issue.

**Comment by taxpon** [comment]
@wing328 Sure. So we need to wait for Swagger Parser updating. Let me know anything I can contribute, thanks.
